packages/now-next/src/index.ts

```diff
@@ -368,18 +385,21 @@ export const build = async ({
   console.log('Installing dependencies...');
   await runNpmInstall(entryPath, ['--prefer-offline'], spawnOpts, meta);
 
-  let realNextVersion: string | undefined;
-  try {
-    realNextVersion = require(resolveFrom(entryPath, 'next/package.json'))
-      .version;
+  // Refetch Next version now that dependencies are installed.
+  // This will now resolve the actual installed Next version,
+  // even if Next isn't in the project package.json
+  nextVersion = await getNextVersion(entryPath);
```

Comment:
This should fail if Next cannot be found using the `required` flag mentioned in review.

Reply:
@Timer 2 lines below this, an error is thrown if `nextVersion` isn't found. If I'm understanding you correctly, adding the `required` flag will duplicate that. I.e. the missing next version is already being handled.

So do you want to keep it like it is or do you want to move the error check right below this into `getNextVersion()`?